### PR TITLE
fix (chatbot): Account for `ai_instructions` in Explore system prompt

### PR DESCRIPTION
**Pull request by ericpgreen2**
**Checklist:**
- [ ] Covered by tests
- [x] Ran it and it works as intended
- [x] Reviewed the diff before requesting a review
- [x] Checked for unhandled edge cases
- [ ] Linked the issues it closes
- [ ] Checked if the docs need to be updated. If so, create a separate Linear DOCS issue
- [x] Intend to cherry-pick into the release branch
- [x] I'm proud of this work!

### CODE REVIEW

File: runtime/completion.go [hunk 1]
```diff
@@ -209,42 +209,39 @@ func (r *Runtime) processExploreDashboardContext(ctx context.Context, instanceID
...
-	timeRangeResult, err := toolService.ExecuteTool(ctx, "query_metrics_view_time_range", map[string]any{
+	timeRangeSummary, err := toolService.ExecuteTool(ctx, "query_metrics_view_time_range", map[string]any{
 		"metrics_view": metricsViewName,
 	})
 	if err != nil {
 		return nil, err
 	}
 
+	// Find instance-wide AI context
+	aiInstructions := ""
```
Comment (k-anshul) [inline runtime/completion.go]:
```suggestion
	var aiInstructions string
```
